test/e2e/features_test.go

```diff
@@ -0,0 +1,81 @@
+package e2e
+
+import (
+	//"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/cucumber/godog"
+	"github.com/cucumber/godog/colors"
+	"github.com/spf13/pflag"
+	ctrl "sigs.k8s.io/controller-runtime"
+	//ctrllog "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
```

Comment:
Avoid new zap dependency? I think we're using klog in our main.go's. Can we use that here too?